desk/app/expose.hoon

```diff
@@ -0,0 +1,2233 @@
+      ;head
+        ;title:"{title}"
+        ;style:"{(trip style)}"
+        ;link(rel "preconnect", href "https://rsms.me");
+        ;link(rel "stylesheet", href "https://rsms.me/inter/inter.css");
+      ::
+        ;meta(charset "utf-8");
+        ;meta(name "viewport", content "width=device-width, initial-scale=1");
+      ::
+        ;meta(name "robots", content "noindex, nofollow, noimageindex");
+      ::
+        ::REVIEW  make sure this is the right/new app id
+        ;meta(property "apple-itunes-app", content "app-id=6451392109");
+        ::NOTE  at the time of writing, android supports no such thing
+      ::
+        ::TODO  could get even smarter about description, preview image, etc
+        ;meta(property "og:title", content title);
+        ;meta(property "twitter:title", content title);
+        ;meta(property "og:site_name", content "Tlon");
+        ;meta(property "og:type", content "article");
+        ;meta(property "og:article:author:username", content (scow %p author.msg));
+      ::
+        ;*  ?~  img
+            :_  ~
+            ;meta(property "twitter:card", content "summary");
+        =/  img=tape  (trip u.img)
+        :~  ;meta(property "twitter:card", content "summary_large_image");
+            ;meta(property "og:image", content img);
+            ;meta(property "twitter:image", content img);
+        ==
+      ::
+        ;*  ?~  aco  ~
+            ?:  =('' nickname.u.aco)  ~
+            :_  ~
+        ;meta(property "og:article:author:first_name", content (trip nickname.u.aco));
+      ==
+    ::
+    ++  footer
+      ;footer
+        ;a(href "https://tlon.io")
+          ;img@"https://tlon.io/icon.svg"(alt "Tlon logo", width "18");
+          ;span
+            ; Powered by Tlon
+          ==
+        ==
+      ==
+    ::
+    ++  chat-prelude
+      ^-  manx
+      ;div.prelude
+        ;div.published
+          ;+  (render-datetime sent.msg)
+        ==
+        ;+  author-node
+      ==
+    ::
+    ++  diary-prelude
+      |=  title=tape
+      ^-  marl
+      :~  ;h1:"{title}"
+          ;div.prelude
+            ;div.published
+              ;+  (render-datetime sent.msg)
+            ==
+            ;+  author-node
+          ==
+      ==
+    ::
+    ++  heap-prelude
+      |=  title=tape
+      ^-  marl
+      =-  ?:  =("" title)  [-]~
+          :-  ;h1:"{title}"
+          [-]~
+      ;div.prelude
+        ;div.published
+          ;+  (render-datetime sent.msg)
+        ==
+        ;+  author-node
+      ==
+    ::
+    ++  author-node
+      ^-  manx
+      =*  author  author.msg
+      ;div.author
+        ;div.avatar
+          ;+
+          ?:  &(?=(^ aco) ?=(^ avatar.u.aco) !=('' u.avatar.u.aco))
+            ;img@"{(trip u.avatar.u.aco)}"(alt "Author's avatar");
+          =/  val=@ux   ?~(aco 0x0 color.u.aco)
+          =/  col=tape  ((x-co:^co 6) val)
+          %.  author
+          %_  sigil
+            size  25
+            icon  &
+            bg    '#'^col
```

Comment:
why do you need the ket here

Reply:
Ket as in shorthand for making a cell, `['#' col]`, rather than `^col` for going to the "next one up".